builder/dockerfile/builder.go

```diff
@@ -173,11 +173,15 @@ func (b *Builder) Cancel() {
 	})
 }
 
-// BuildFromConfig will do build directly from parameter 'changes', which comes
-// from Dockerfile entries, it will:
-// - call parse.Parse() to get AST root from Dockerfile entries
-// - do build by calling builder.dispatch() to call all entries' handling routines
-// TODO: remove?
+// BuildFromConfig will do build directly from parameter 'changes',
```

Comment:
I suggest `BuildFromConfig builds directly from changes, treating it as if it were the contents of a Dockerfile.`